packages/steiger/src/models/config/transform-globs.ts

```diff
@@ -0,0 +1,50 @@
+import { posix, sep } from 'node:path'
+import { Config } from '@steiger/types'
+
+import { isConfigObject, isConfiguration } from './raw-config'
+
+function convertRelativeGlobsToAbsolute(rootPath: string, globs: Array<string>) {
+  function composeAbsolutePath(root: string, glob: string) {
+    // Remove '/'. The root has platform-specific separators
+    const segmentsOfRoot = root.slice(1).split(sep)
+
+    return `/${posix.join(...segmentsOfRoot, glob)}`
+  }
+
+  return globs.map((glob) => (glob.startsWith('.') ? composeAbsolutePath(rootPath, glob) : glob))
```

Comment:
issue (non-blocking): I think we should be checking if the glob starts with `/` to tell if it's absolute or not. Otherwise we will not convert a glob like `"src/entities/**"`, which is a valid relative path, just the `./` is omitted.

suggestion: let's check if the path is absolute with [`path.isAbsolutePath`](https://nodejs.org/api/path.html#pathisabsolutepath) and add this case to the tests